compiler/src/Language/Bond/Codegen/Cpp/Grpc_h.hs

```diff
@@ -49,11 +49,11 @@ grpc_h export_attribute cpp file imports declarations = ("_grpc.h", [lt|
 #pragma warning (disable: 4100 4267)
 #endif
 
-#include <grpc++/impl/codegen/channel_interface.h>
```

Comment:
Missing updated golden test files for `gbc`.

Reply:
Thanks. Fixed in 0544f7a7.